pkg/manager/member/pd_member_manager.go

```diff
@@ -713,7 +713,15 @@ func getPDConfigMap(tc *v1alpha1.TidbCluster) (*corev1.ConfigMap, error) {
 		config.Dashboard.TiDBCertPath = path.Join(tidbClientCertPath, corev1.TLSCertKey)
 		config.Dashboard.TiDBKeyPath = path.Join(tidbClientCertPath, corev1.TLSPrivateKeyKey)
 	}
-
+	if tc.Spec.TiFlash != nil {
```

Comment:
If TiFlash is enabled and EnablePlacementRules is not set explicitly, enable it.

Reply:
Can we add a document to explain why EnablePlacementRules is necessary for tiflash

Reply:
Updated